### PR TITLE
Add new Press Center page

### PR DESCRIPTION
**Pull request by carolromero**

### CODE REVIEW

**Comment by andreslucena** [comment]
Deploy preview for *decidim-website* ready!

Built with commit 6286a10f0994e1b17d3f3f2346a15056e23b9118

https://deploy-preview-239--decidim-website.netlify.com

File: source/localizable/press.ca.html.erb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  <li><a href="https://www.rbb-online.de/doku/b/besser-geht-immer/besser-geht-immer.html">Besser geht immer. Astrid Frohloff trifft kreative Bürokraten</a> RBB</li>
+</ul>
+
+<h2>FAQs</h2>
+<p>Fes clic <a href="https://decidim.org/ca/faqs/">aquí</a>.</p>
+
+<h2>Recursos de comunicació</h2>
+
+<ul>
+  <li><a href="https://decidim.org/pdf/identity-manual-ca.pdf">Logos</a></li>
```
Comment (andreslucena) [inline source/localizable/press.ca.html.erb]: it's better to use relative URLs here like
`   <li><a href="/pdf/identity-manual-ca.pdf">Logos</a></li>`

Comment (andreslucena) [reply source/localizable/press.ca.html.erb]: the same on FAQ, triptic, etc.

Comment (carolromero) [reply source/localizable/press.ca.html.erb]: Sorry! Done

**Pull request merged**